templates/controller-cert-manager-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.controller.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "consul.fullname" . }}-webhook-cert-manager-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
```

Comment:
component?

Reply:
Honestly, it wasn't mentioned in our other config maps and hence the skip